public/app/features/logs/components/LogRowContext.tsx

```diff
@@ -73,6 +92,13 @@ const getLogRowContextStyles = (theme: GrafanaTheme2, wrapLogMessage?: boolean)
       align-items: center;
       background: ${theme.colors.background.canvas};
     `,
+    datasourceUi: css`
+      height: ${datasourceUiHeight}px;
```

Comment:
i would maybe increase this to stop the UI being so cramped. i think `+ 30px` makes it look better, but this is just preference

Reply:
If you are fine with that, I would consider this when we do the redesign in Q1 as I already ran into situations where the current total height of the LogContext was more than my screen allows (on a laptop though).